common/wireaddr.c

```diff
@@ -37,11 +37,13 @@ bool fromwire_wireaddr(const u8 **cursor, size_t *max, struct wireaddr *addr)
 	case ADDR_TYPE_TOR_V3:
 		addr->addrlen = TOR_V3_ADDRLEN;
 		break;
+#if EXPERIMENTAL_FEATURES /* BOLT7 DNS RFC #911 */
 	case ADDR_TYPE_DNS:
 		addr->addrlen = fromwire_u8(cursor, max);
 		memset(&addr->addr, 0, sizeof(addr->addr));
 		addr->addr[addr->addrlen] = 0;
 		break;
+#endif
 	case ADDR_TYPE_WEBSOCKET:
```

Comment:
This is too much, IMHO.  Disable option parsing, sure, but don't disable it internally!

Reply:
Okay, also lookup abd connect to DNS announced nodes without experimental then?
Or skip DNS announcements without experimental?